doc/services/portability/posix/overview/index.rst

```diff
@@ -0,0 +1,129 @@
+.. _posix_overview:
+
+Overview
+########
+
+The Portable Operating System Interface (POSIX) is a family of standards specified by the `IEEE Computer Society`_ for maintaining compatibility between operating systems. Zephyr
+implements a subset of the standard POSIX API specified by `IEEE 1003.1-2017`_ (also known as POSIX-1.2017).
+
+..  figure:: posix.svg
+    :align: center
+    :alt: POSIX Support in Zephyr
+
+    POSIX support in Zephyr
+
+.. note::
+    This page does not document Zephyr's :ref:`POSIX architecture<Posix arch>`, which is used to
+    run Zephyr as a native application under the host operating system for prototyping,
+    test, and diagnostic purposes.
+
+With the POSIX support available in Zephyr, an existing POSIX conformant
```

Comment:
I would like to avoid modifying existing content if possible, and was hoping to focus mainly on structure. That's mainly because @kartben wanted a refresh of the posix docs for v3.5.0. The changes in this PR should be able to be backported.

https://github.com/zephyrproject-rtos/zephyr/blob/26f30db8adabe40983ca2501e1c866dc7ebe2d5c/doc/services/portability/posix.rst?plain=1#L11